lib/box-client.js

```diff
@@ -386,7 +386,12 @@ BoxClient.prototype.setIPs = function(ips) {
 	var validIPs = ips.filter(function(ipString) {
 		return isIP(ipString);
 	});
-	this.setCustomHeader(HEADER_XFF, validIPs.join(', '));
+	validIPs = validIPs.join(', ');
```

Comment:
I mean, you should be able to do something like `var validIPs = ips.filter(ipString => isIP(ipString)).join(', ');` as a one-liner